src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -230,13 +208,13 @@ public enum ChunkVersions : byte {
 				_chunkHeader = ReadHeader(reader.Stream);
 				Log.Debug("Opened completed {chunk} as version {version}", _filename, _chunkHeader.Version);
 				if (_chunkHeader.Version != (byte)ChunkVersions.Unaligned &&
-				    _chunkHeader.Version != (byte)ChunkVersions.Aligned)
+					_chunkHeader.Version != (byte)ChunkVersions.Aligned)
 					throw new CorruptDatabaseException(new WrongFileVersionException(_filename, _chunkHeader.Version,
 						CurrentChunkVersion));
 
-				if (_chunkHeader.Version != (byte)ChunkVersions.Aligned && _unbuffered) {
+				if (_chunkHeader.Version != (byte)ChunkVersions.Aligned) {
```

Comment:
This will prevent ES from starting up with chunk files made in ES v3.x which haven't been scavenged on v4 or v5 yet. 

If this is intentional, we should remove the check above that checks for either Unaligned or Aligned chunk versions, and be aware that this introduces an upgrade step

Reply:
cc: @condron

Reply:
Which checks should we we remove above?

Reply:
```csharp
if (_chunkHeader.Version != (byte)ChunkVersions.Unaligned && 
    _chunkHeader.Version != (byte)ChunkVersions.Aligned)	
		throw new CorruptDatabaseException(new WrongFileVersionException(_filename, _chunkHeader.Version, CurrentChunkVersion));
```
At line [211](https://github.com/EventStore/EventStore/pull/2599/files/b3351c7c2bcbcfde11a1ab3d7cb5c559b4eaa901#diff-9906bfb99c1b4f17ed2f491ac6a107caR211) there's a check to ensure that the chunk is either unaligned or aligned

Reply:
@hayley-jean  Added issue in zenhub for the review on this.